site/src/components/SearchField/SearchField.tsx

```diff
@@ -0,0 +1,58 @@
+import { useTheme } from "@emotion/react";
+import CloseOutlined from "@mui/icons-material/CloseOutlined";
+import SearchOutlined from "@mui/icons-material/SearchOutlined";
```

Comment:
Nit: could we start naming our icon imports as `nameIcon`? I feel like it makes the code a little easier to read, especially for people who aren't super familiar with MUI